cmd/nodecmd/loadtest.go

```diff
@@ -264,9 +284,108 @@ func createLoadTest(_ *cobra.Command, args []string) error {
 		return err
 	}
 	ux.Logger.PrintToUser("Load test successfully run!")
+
 	return nil
 }
 
+func getDeployedSubnetInfo() (string, string, error) {
+	sidecarFile := filepath.Join(app.GetSubnetDir(), subnetName, constants.SidecarFileName)
+	var sidecar models.Sidecar
+	if _, err := os.Stat(sidecarFile); err == nil {
+		// read in sidecar file
+		jsonBytes, err := os.ReadFile(sidecarFile)
+		if err != nil {
+			return "", "", fmt.Errorf("failed reading file %s: %w", sidecarFile, err)
+		}
+		err = json.Unmarshal(jsonBytes, &sidecar)
+		if err != nil {
+			return "", "", fmt.Errorf("failed unmarshaling file %s: %w", sidecarFile, err)
+		}
+	}
+	if sidecar.Networks != nil {
+		model, ok := sidecar.Networks["Devnet"]
+		if ok {
+			if model.SubnetID != ids.Empty && model.BlockchainID != ids.Empty {
+				return model.SubnetID.String(), model.BlockchainID.String(), nil
+			}
+		}
+	}
+	return "", "", fmt.Errorf("unable to find deployed Devnet info at cluster_config.json")
+}
+
+func createClusterYAMLFile(clusterName, subnetID, chainID string, separateHost *models.Host) error {
+	clusterYAMLFilePath := filepath.Join(app.GetAnsibleInventoryDirPath(clusterName), constants.ClusterYAMLFileName)
+	if _, err := os.Stat(clusterYAMLFilePath); err == nil {
+		if err = os.Remove(clusterYAMLFilePath); err != nil {
+			return err
+		}
+	}
+	yamlFile, err := os.OpenFile(clusterYAMLFilePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, constants.WriteReadReadPerms)
+	if err != nil {
+		return err
+	}
+	defer yamlFile.Close()
+
+	enc := yaml.NewEncoder(yamlFile)
+
+	clusterInfoMap := make(map[string]string)
+	clustersConfig := models.ClustersConfig{}
+	if app.ClustersConfigExists() {
+		clustersConfig, err = app.LoadClustersConfig()
+		if err != nil {
+			return err
+		}
+	}
+	clusterConf := clustersConfig.Clusters[clusterName]
+	if err := checkCluster(clusterName); err != nil {
+		return err
+	}
+	validatorCount := 0
+	apiNodeCount := 0
+	for _, cloudID := range clusterConf.GetCloudIDs() {
+		validatorIPEnvPrefix := "IP_VALIDATOR_"
```

Comment:
We may want to properly nest these fields like:
```yaml
apis:
  - name: XXX
    ip: YYY
  - name: ZZZ
    ip: MMM
```

Reply:
Turns out Golang doesn't work well with the current format on the parser side (requires unmarshaling to a struct): https://pkg.go.dev/gopkg.in/yaml.v2@v2.4.0

Reply:
in the meantime, will unmarshal to: `obj := make(map[string]interface{})`